From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: gellert <gellert.hegyi@around.co>
Date: Sat, 16 May 2020 22:28:28 +0200
Subject: disables hidden occluded

This patch disables the background throttling in the renderer completely.

Signed-off-by: gellert <gellert.hegyi@around.co>

diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
index 0a78c70d43791c5c975ac6754651897a77423bff..b3499cd7eff2444444a358bd9712c6066e649331 100644
--- a/content/browser/web_contents/web_contents_impl.cc
+++ b/content/browser/web_contents/web_contents_impl.cc
@@ -2173,7 +2173,7 @@ void WebContentsImpl::WasShown() {
 
 void WebContentsImpl::WasHidden() {
   TRACE_EVENT0("content", "WebContentsImpl::WasHidden");
-  UpdateVisibilityAndNotifyPageAndView(Visibility::HIDDEN);
+  // UpdateVisibilityAndNotifyPageAndView(Visibility::HIDDEN);
 }
 
 bool WebContentsImpl::HasRecentInteractiveInputEvent() {
@@ -2230,7 +2230,7 @@ void WebContentsImpl::SetPrimaryMainFrameImportance(
 
 void WebContentsImpl::WasOccluded() {
   TRACE_EVENT0("content", "WebContentsImpl::WasOccluded");
-  UpdateVisibilityAndNotifyPageAndView(Visibility::OCCLUDED);
+  // UpdateVisibilityAndNotifyPageAndView(Visibility::OCCLUDED);
 }
 
 Visibility WebContentsImpl::GetVisibility() {
